src/current/v23.2/read-committed.md

```diff
@@ -0,0 +1,936 @@
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT * FROM kv AS OF SYSTEM TIME '2023-11-09 21:22:10' WHERE k = 1;
+~~~
+
+~~~
+  k | v
+----+----
+  1 | 3
+~~~
+
+{{site.data.alerts.callout_info}}
+While concurrent `READ COMMITTED` transactions can have their writes overwritten, **uncommitted** writes in `READ COMMITTED` transactions cannot be overwritten.
+{{site.data.alerts.end}}
+
+#### Write skew anomaly
+
+The following sequence of operations is possible under `READ COMMITTED` isolation: 
+
+1. Transaction `A` reads row `R` in table `T` at timestamp `1`.
+1. Transaction `B` writes to row `R` in table `T` and commits at timestamp `2`.
+1. Transaction `A` writes to another row in table `T` and commits at timestamp `3`.
+
+The value of `R` has changed while transaction `A` is open. However, `A` can still write and commit instead of aborting, since `READ COMMITTED` transactions do not require serializability. This does not inherently create an anomaly, but is the basis of potential *write skew anomalies* where two concurrent transactions each read values that the other subsequently updates.
+
+{{site.data.alerts.callout_info}}
+For details on why this is allowed, see [Read refreshing]({% link {{ page.version.version }}/architecture/transaction-layer.md %}#write-skew-tolerance).
+{{site.data.alerts.end}}
+
+##### Example: Write skew anomaly
+
+For an example of how a write skew anomaly can occur, see [Demonstrate interleaved statements in `READ COMMITTED` transactions](#demonstrate-interleaved-statements-in-read-committed-transactions).
+
+## Locking reads
+
+To reduce the occurrence of [concurrency anomalies](#concurrency-anomalies) in `READ COMMITTED` isolation, you can strengthen the isolation of individual reads by using [`SELECT ... FOR UPDATE`]({% link {{ page.version.version }}/select-for-update.md %}) or [`SELECT ... FOR SHARE`]({% link {{ page.version.version }}/select-for-update.md %}) to issue *locking reads* on specific rows. Locking reads behave similarly to [writes]({% link {{ page.version.version }}/architecture/transaction-layer.md %}#write-intents): they lock qualifying rows to prevent concurrent writes from modifying them until the transaction commits. Conversely, if a locking read finds that a row is exclusively locked by a concurrent transaction, it waits for the other transaction to commit or rollback before proceeding. A locking read in a transaction will always have the latest value of a row when the transaction commits.
+
+The clause used with the `SELECT` statement determines the *lock strength* of a locking read:
+
+{% include {{ page.version.version }}/sql/select-lock-strengths.md %}
+
+### When to use locking reads
+
+Use locking reads in your application if certain `READ COMMITTED` transactions must guarantee data correctness in [high-contention]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#transaction-contention) scenarios.
```

Comment:
nit: I find this sentence a little confusing. Maybe instead of "must guarantee data correctness in high-contention scenarios" something like "must guarantee data invariants that would be violated by concurrency anomalies"?

(But that's sort of tautological... eh I dunno if it's actually better)

Reply:
How about this: "must guarantee that the data they access will not be changed by intermediate writes."